scheduler/src/cook/mesos/api.clj

```diff
@@ -754,10 +748,10 @@
 
 (defn- instance->instance-map
   "Converts the instance to a map of relevant fields that will be returned to the caller"
-  [db fid instance]
+  [db framework-id retrieve-url-path-fn instance]
```

Comment:
We could pass the `cache` reference instead and then invoke `retrieve-url-path` directly. Is that preferred?